dashboard/src/scripts/config.js

```diff
@@ -1,4 +1,4 @@
-var dataBaseDir = 'https://cdn.rawgit.com/okfn/spd-data-example/master/data';
+var dataBaseDir = 'https://cdn.rawgit.com/okfn/spd-data-uk-all/master/data';
```

Comment:
@hdurand the base repository should refer to the example data, not to specific instance data. 

If the example data needs updating, please do that.

Specific instance data data should only be configured for the repo that serves the instance, as here: https://github.com/okfn/spd-uk-cabinet-office